### PR TITLE
Feature/single pretix date

### PR DESCRIPTION
**Pull request by martinyde**
- Javascript for locking down Pretix to date field.

https://jira.itkdev.dk/browse/HRIN-448

### CODE REVIEW

File: web/modules/custom/hoeringsportal_public_meeting/assets/js/single-date.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require('../css/single-date.css')
+
+document.querySelectorAll('.date-to-element input.form-date')[0].disabled = true
+let tableRow = document.querySelectorAll('#field-pretix-dates-values tr')
+for (var i = 0; i < tableRow.length; i++) {
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_public_meeting/assets/js/single-date.js]: [`Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) and [`Array.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) are your friends: 

```js
const tableRows = Array.from(document.querySelectorAll('#field-pretix-dates-values tr'))
tableRows.forEach(tableRow => { … })
```

File: web/modules/custom/hoeringsportal_public_meeting/assets/js/single-date.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require('../css/single-date.css')
+
+document.querySelectorAll('.date-to-element input.form-date')[0].disabled = true
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_public_meeting/assets/js/single-date.js]: Use [`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) when you only want the first element.

File: web/modules/custom/hoeringsportal_public_meeting/assets/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+  // The project directory where all compiled assets will be stored.
+  .setOutputPath('public/build/')
+
+  // The public path used by the web server to access the previous directory.
+  .setPublicPath('/build')
+
+  // Will create public/build/app.js and public/build/app.css.
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_public_meeting/assets/webpack.config.js]: Maybe we should just remove this comment …

**Comment by martinyde** [comment]
Putting this on hold for now

**Pull request closed without merging**